shub_image/push.py

```diff
@@ -60,11 +64,9 @@ def _execute_push_login(client, image, username, password, email):
     """Login if there're provided credentials for the registry"""
     components = image.split('/')
     registry = components[0] if len(components) == 3 else None
-    if password is None:
-        # Missing password leads to auth request to registry authentication service
-        # without 'account' query parameter which breaks login procedure.
-        password = ' '
-    resp = client.login(username=username, password=password,
+    # Missing password leads to auth request to registry authentication service
+    # without 'account' query parameter which breaks login procedure.
+    resp = client.login(username=username, password=password or ' ',
```

Comment:
above I checked `password is None` because password can be empty string if it was specified. and if it was explicitly set - I think you don't want to override it silently, right?